secret-manager/cleaner.sh

```diff
@@ -91,6 +91,21 @@ else
   exit 1
 fi
 
+response=$(kubectl delete sa/xkops-secret-sa --namespace xkops 2>&1)
+# possible responses:
+# serviceaccount "xkops-secret-sa" deleted
+# Error from server (NotFound): serviceaccounts "xkops-secret-sa" not found
+
+if [[ $response == *"deleted"* ]]; then
+  echo "Successfully deleted xkops-secret-sa service account from the cluster."
+elif [[ $response == *"Error from server (NotFound)"*  ]]; then
+  echo "xkops-secret-sa service account has already been removed from the cluster."
+else
+  echo "$response"
+  echo "Failed to remove xkops-secret-sa service account from the cluster. Exiting..."
+  exit 1
+fi
+
 response=$(kubectl delete namespace xkops 2>&1)
```

Comment:
`kubectl delete pods -n xkops --all --grace-period 0 --force` needs to be run first then delete xkops namespace to handle pods stuck in terminating state